packages/neuron-ui/src/styles/mixin.scss

```diff
@@ -175,8 +175,12 @@
     svg {
       g,
       path {
-        fill: $hoverColor
+        fill: $hoverColor;
       }
     }
   }
 }
+
+@mixin cursor-mode {
```

Comment:
Is it necessary to define one property with `mixin`? I think `cursor: pointer` is clear to replace `@include cursor-mode;`